org.eclipse.lyo.oslc4j.core/src/main/java/org/eclipse/lyo/oslc4j/core/model/ResourceShapeFactory.java

```diff
@@ -362,7 +365,7 @@ private static Occurs getDefaultOccurs(final Class<?> type) {
 		}
 	}
 
-	protected static void validateSetMethodExists(final Class<?> resourceClass, final Method getMethod) throws OslcCoreMissingSetMethodException {
+	static void validateSetMethodExists(final Class<?> resourceClass, final Method getMethod) throws OslcCoreMissingSetMethodException {
```

Comment:
The access specifier should not be "default" This method is access from Shacl project.  [ShaclShapeFactory.Java Class](https://github.com/eclipse/lyo.core/blob/master/org.eclipse.lyo.shacl/src/main/java/org/eclipse/lyo/shacl/ShaclShapeFactory.java#L319)

Reply:
"package" is more open than "protected" or am I missing something?

can't recall why did I have a need to modify it

Reply:
The ShaclShapeFactory class is not in the same "package" as ResourceShapeFactory class.  ResourceShapeFactory class is extended from ShaclShapeFactory class that is in another project/package.

"Default" is not accessible in different package subclass. Hence , we need protected or public. 

See this: https://www.geeksforgeeks.org/access-modifiers-java/